src/UCTNode.cpp

```diff
@@ -291,7 +291,7 @@ void UCTNode::accumulate_eval(float eval) {
 
 UCTNode* UCTNode::uct_select_child(int color) {
     UCTNode* best = nullptr;
-    auto best_value = -1000.0;
+    auto best_unvisited_psa = -1000.0;
```

Comment:
Maybe instead of hardcoding the -1000 in several places we should use <limit> to get some minimal double value and use a named constant/constexpr.

Reply:
I see why you say that. Just within the scope of this function?

Reply:
Sure.